packages/test/test-service-load/src/runner.ts

```diff
@@ -268,8 +265,18 @@ async function runnerProcess(
 				);
 			}
 
+			const closeDisposeProm = new Promise<boolean>((resolve) => {
```

Comment:
@anthony-murphy @vladsud This is a solution I think makes sense, but more work is still needed as the code still hangs in the same spot (doesn't allow this `closeDisposeProm` to realize a closure has happened)

Reply:
I don't really understand the new model. In this case, the stress workload wants a signal to stop doing work, this was previously the dispose of the runtime and data stores. now it appears close doesn't signal the runtime, maybe it goes readonly, but i'm not sure this is enough of a signal to stop doing work. I can imagine a real scenario where a container connects to some external service, like aug loop to pass along or get data, how will a container like this know that it should stop doing work like that with the new application model? Another case is leader election, the leader needs to know it is no longer the leader, and non-leaders would want to know that no new leader will be elected, as no new changes will come.

Reply:
if we think the container runtime shouldn't know if it is closed, and it should keep doing or attempting to do any work it deems necessary, then we don't need any changes. However, i would question if this is good solution for customers.

if the container runtime can know it is in a closed state, than the fix here should be in the load test datastore, and it should stop attempting to do work when in that state, specifically, it should stop waiting for task manager events, and exit its main run loop.